scalafmt-tests/src/test/resources/scala3/Extension.stat

```diff
@@ -167,3 +167,19 @@ extension [A, B](a: A) /* foo */ (using
 )
   /* baz */
   def a = 1
+<<< multiple using clauses
```

Comment:
do you want to add variants of this test with different values of `implicitParamListModifierForce` and `implicitParamListModifierPrefer`?

Reply:
I think we should be fine, looks like everything is working irrelevant which param list it is.

Reply:
I added some more tests, it seems to work ok as far as I can tell.

Reply:
lovely, it really does work! :)